lib/ui/book_screen/book_screen.dart

```diff
@@ -148,11 +149,13 @@ class BookScreen extends StatelessWidget {
   }
 
   String _generateReadingTime({
-    required DateTime startDate,
-    required DateTime finishDate,
+    DateTime? startDate,
+    DateTime? finishDate,
     required BuildContext context,
+    ReadingTime? readingTime,
   }) {
-    final diff = finishDate.difference(startDate).inDays.toString();
+    if (readingTime != null) return readingTime.toString();
```

Comment:
I you used build--in DateTime this line could be:
if (readingTime != null) return '${readingTime.inDays} ${LocaleKeys.days.tr()}';